tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -719,6 +881,11 @@ decompress_chunk_explain(CustomScanState *node, List *ancestors, ExplainState *e
 		{
 			ExplainPropertyBool("Sorted merge append", chunk_state->sorted_merge_append, es);
 		}
+
+		if (es->analyze && chunk_state->using_bulk_decompression)
+		{
+			ExplainPropertyBool("Bulk Decompression", true, es);
+		}
```

Comment:
```suggestion
		if (es->verbose || es->format != EXPLAIN_FORMAT_TEXT)
		{
			ExplainPropertyBool("Bulk Decompression", chunk_state->using_bulk_decompression, es);
		}
```
Hmm i would change the logic for including this since it is the default.

Reply:
Changed.